commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/TwoSidedDrive.scala

```diff
@@ -8,6 +8,28 @@ import squants.{Acceleration, Dimensionless, Each, Length, Time, Velocity}
 import squants.motion.{MetersPerSecond, MetersPerSecondSquared}
 import squants.space.Meters
 
+trait TwoSidedDriveHardware extends UnicycleHardware {
+  val leftVelocity: PeriodicSignal[Velocity]
+  val rightVelocity: PeriodicSignal[Velocity]
+
+  val forwardVelocity: PeriodicSignal[Velocity] =
+    leftVelocity.zip(rightVelocity).map(t => (t._1 + t._2) / 2)
+}
+
+trait TwoSidedDriveProperties extends UnicycleProperties {
+  val maxLeftVelocity: Velocity
+  val maxRightVelocity: Velocity
+  val leftControlGains: VelocityGains
```

Comment:
Yeah, that's correct for taking the two gains. Also generally in case we want to use it in some weird situation where gear ratios are different or something.

Unicycle drivetrains take a props with forward control gains, since unicycles have forward/backward and left/right control. But since we're essentially controlling the forward/backward speed through the left/right gains, we disable forward/backward control.